freqtrade/optimize/backtesting.py

```diff
@@ -275,8 +275,34 @@ def load_bt_data_detail(self) -> None:
                 if pair not in self.exchange._leverage_tiers:
                     unavailable_pairs.append(pair)
                     continue
-                self.futures_data[pair] = funding_rates_dict[pair].merge(
-                    mark_rates_dict[pair], on='date', how="inner", suffixes=["_fund", "_mark"])
+
+                if (pair in mark_rates_dict
+                        and len(funding_rates_dict[pair]) == 0
+                        and "futures_funding_rate" in self.config):
+                    mark_rates_dict[pair]["open_fund"] = self.config.get('futures_funding_rate')
+                    mark_rates_dict[pair]["close_fund"] = 0.0
+                    mark_rates_dict[pair]["high_fund"] = 0.0
+                    mark_rates_dict[pair]["low_fund"] = 0.0
+                    mark_rates_dict[pair]["volume_fund"] = 0.0
```

Comment:
You can safely ignore these columns - only `open_fund` and `open_mark` are actually relevant to this dataframe.

Reply:
OK we can remove those - i thought they might be necessary for something downstream so left them in

Reply:
And done